ha/util/mock_health_event_publisher.py

```diff
@@ -0,0 +1,100 @@
+    """
+    # TODO: This is temporary code till this is avalable in HA.
+    num_of_cvgs = Conf.get(_index, f'node>{node_id}>storage>num_cvg')
+    disk_ids = []
+    for cvg in range(num_of_cvgs):
+        num_of_data_disks = Conf.get(_index, f'node>{node_id}>storage>cvg[{cvg}]>devices>num_data')
+        num_of_metadata_disks = Conf.get(_index, f'node>{node_id}>storage>cvg[{cvg}]>devices>num_metadata')
+        [ disk_ids.append(Conf.get(_index, f'node>{node_id}>storage>cvg[{cvg}]>devices>data[{dt_disk}]')) for dt_disk in range(num_of_data_disks)]
+        [ disk_ids.append(Conf.get(_index, f'node>{node_id}>storage>cvg[{cvg}]>devices>metadata[{md_disk}]')) for md_disk in range(num_of_metadata_disks)]
+    print(disk_ids)
+
+def get_cvgs(conf_store: ConftStoreSearch, node_id: str) -> None:
+    """
+    Fetches cvg ids using ConfStore search API and displays the result
+
+    Args:
+    conf_store: ConftStoreSearch object
+    node_id: machine_id value
+    """
+    # TODO: This is temporary code till this is avalable in HA.
+    cvg_count = Conf.get(_index, f'node>{node_id}>storage>num_cvg')
+    print([Conf.get(_index, f'node>{node_id}>storage>cvg[{cvg}]>name') for cvg in range(cvg_count)])
+
+def publish(conf_store=None, node_id=None):
+    print('inside publish')
+
+FUNCTION_MAP = {
+                '-gdt' : get_data_nodes, '--get-data-nodes': get_data_nodes,
+                '-gs' : get_server_nodes, '--get-server-nodes': get_server_nodes,
+                '-gd' : get_disks, '--get-disks': get_disks,
+                '-gc' : get_cvgs, '--get-cvgs': get_cvgs,
+                '-p' : publish, '--publish': publish
+                }
+
+my_parser = argparse.ArgumentParser(prog='health_event_publisher',
+                                    usage='%(prog)s [options]',
+                                    description='Helps in publishing the mock health event',
+                                    epilog='Hope it is useful! :)')
+
+get_options = my_parser.add_mutually_exclusive_group(required=True)
+get_options.add_argument('-gd', '--get-disks', action='store_true', help='Get the DISK IDs')
+get_options.add_argument('-gc', '--get-cvgs', action='store_true', help='Get the CVG IDs')
+get_options.add_argument('-gdt', '--get-data-nodes', action='store_true', help='Get the data nodes')
+get_options.add_argument('-gs', '--get-server-nodes', action='store_true', help='Get the server nodes')
+get_options.add_argument('-p', '--publish', action='store_true', help='Publish the message')
+
+my_parser.parse_args()
+
+if __name__ == '__main__':
+    Conf.init()
+    file_to_load = f'{config_file_format}://{cluster_config_file}'
+    Conf.load(_index, file_to_load)
+    _conf_store = ConftStoreSearch(conf_store_req=False)
+    # To fetch disk or cvg id, node id is required. So, one of the node_id
+    # will be picked from the list which we will get from get_data_nodes function
+    # and ids of disk and cvg assosciated with that node will be displayed
```

Comment:
How do we get a list of disk ids associated with other nodes?

Reply:
You will need to specify that specific node id. If we want to do that then we need to accept one more argument for this

Reply:
Done.